docs/technical-reference/makingBackups.rst

```diff
@@ -0,0 +1,26 @@
+Creating Backups and Copy to Host
+======================================================================
+
+Create Backups
+----------------------------------------------------------------------
+
+To create a backup of the postgres container, use the command:
+    ::
+
+        docker-compose -f production.yml (exec |run --rm) postgres backup
+
+If you are on windows and the above command doesn't work, delete the headings in the script files you are calling, rebuild and run the server, and then use the command:
+    ::
+
+        docker exec -it llemr_postgres_1 /bin/bash usr/local/bin/backup
```

Comment:
Are you able to confirm that deleting the headings is necessary? If it is necessary, would it be valid to delete the headings across the board? Having two different versions of the files for different operating systems is not something that I want to have happen

Reply:
Where exactly is `llemr_postgres_1` coming from? This name seems hardcoded, and I wouldn't be surprised if it was different from one person's computer to another. You should detail how to get that name